src/grpc_requests/aio.py

```diff
@@ -122,10 +125,14 @@ def channel(self):
         return self._channel
 
     @classmethod
-    def get_by_endpoint(cls, endpoint, **kwargs):
+    async def get_by_endpoint(
```

Comment:
This would be an api breaking change. You may need to refactor to a different pattern instead of replacing this one.

Reply:
Yes, you're right. I have to think more about this 👍 , thanks for feedback.

Reply:
Thanks for taking the time to put the PR together - when you've got your next take together, we'll be happy to look it over.